action/protocol/execution/testdata-istanbul/datacopy.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.8.3;
+
+contract Datacopy {
+    function dataCopy() public view returns (bool) {
+        bytes memory arr = new bytes(3);
+        arr[0] = 0x11;
+        arr[1] = 0x22;
+        arr[2] = 0x00;
```

Comment:
with the fix, we should have [0x11, 0x22, 0x00] after the copy, can we check that?
like `return arr[0] = 0x11 && arr[1] = 0x22 && arr[2] = 0x00`

Reply:
Unable to catch the return value unless the code in core/vm/evm.go is modified